flow/transactions/send/send.go

```diff
@@ -31,17 +31,19 @@ import (
 )
 
 type Config struct {
-	Signer  string `default:"service" flag:"signer,s"`
+	Args    string `default:"" flag:"args" info:"arguments to pass to transaction"`
```

Comment:
Same as above, maybe mention in the info that the option requires input in JSON-Cadence format